vars/checksPublishResults.groovy

```diff
@@ -0,0 +1,240 @@
+import com.cloudbees.groovy.cps.NonCPS
+
+import com.sap.piper.ConfigurationLoader
+import com.sap.piper.ConfigurationMerger
+
+import groovy.transform.Field
+
+@Field def STEP_NAME = 'checksPublishResults'
+
+/**
+ * checksPublishResults
+ *
+ * @param others document all parameters
+ */
+def call(Map parameters = [:]) {
+    handlePipelineStepErrors (stepName: STEP_NAME, stepParameters: parameters) {
+        def script = parameters.script
+        if (script == null)
+            script = [commonPipelineEnvironment: commonPipelineEnvironment]
+        prepareDefaultValues script: script
+
+        Map configurationKeys = [
+            'aggregation': [
+                'active': null,
+                'healthy': null,
+                'unHealthy': null,
+                'thresholds': [
+                    'fail': ['all': null,'low': null,'normal': null,'high': null],
```

Comment:
Should't this long, long structure come from the default configuration? IMO we have the default configuration for putting configuration out of the steps. Now we have have that default configuration in the step.